app/Domain/Goalcanvas/Templates/canvasDialog.blade.php

```diff
@@ -0,0 +1,182 @@
+@extends($layout)
+@section('content')
+
+@php
+
+$canvasName = 'goal';
+
+$canvasItem = $tpl->get('canvasItem');
```

Comment:
`$tpl->get` is not required when using blade. You can access the variables directly (`$canvasItem`)